### PR TITLE
Implement 'nock' in Language Translator Tests

### PR DESCRIPTION
**Pull request by dfavretto**
Nock was implemented in the Language Translator Tests and the sinon package was removed.
* Mocks were removed from the code and individual files for each test were created containing the mocked responses
* The methods for consuming sinon were removed
* Nock was implemented in order to run the tests in isolation
* Environment variable was created to control the use of mocks

### CODE REVIEW

File: libraries/botbuilder-ai/tests/TestData/LanguageTranslator/should_bypass_calling_service_in_middleware_for_non-message_activities..json [hunk 1]
```diff
@@ -0,0 +1 @@
+""
```
Comment (JuanAr) [inline libraries/botbuilder-ai/tests/TestData/LanguageTranslator/should_bypass_calling_service_in_middleware_for_non-message_activities..json]: This filename has two dots before extension

File: libraries/botbuilder-ai/tests/languageTranslator.test.js [hunk 2]
```diff
@@ -15,39 +21,45 @@ class TestContext extends TurnContext {
...
 }
 
+function getFilePath(testName) {
+    var filename = testName.replace(/ /g, '_');
+    filename = filename.replace(/"/g, '');
+    return `${ __dirname }/TestData/LanguageTranslator/${ filename }.json`;
+}
+
 describe('LanguageTranslator', function () {
     this.timeout(10000);
```
Comment (JuanAr) [inline libraries/botbuilder-ai/tests/languageTranslator.test.js]: If possible, please remove this line.

File: libraries/botbuilder-ai/tests/TestData/LanguageTranslator/should_bypass_calling_service_in_middleware_for_non-message_activities..json (hunk 1 above)
Comment (dfavretto) [reply libraries/botbuilder-ai/tests/TestData/LanguageTranslator/should_bypass_calling_service_in_middleware_for_non-message_activities..json]: That may be due to the Test name itself having a dot at the end of the text, and the file names were generated automatically. I'll correct that in both places, thanks.

**Comment by dfavretto** [comment]
Closing PR per [#370](https://github.com/Microsoft/botbuilder-js/pull/370)

**Pull request closed without merging**